CMakeLists.txt

```diff
@@ -223,10 +225,10 @@ else()
 	check_and_add_flag(VISIBILITY_INLINES_HIDDEN -fvisibility-inlines-hidden)
 	check_and_add_flag(VISIBILITY_HIDDEN -fvisibility=hidden)
 
-	check_c_compiler_flag(-fomit-frame-pointer FLAG_C_FOMIT_FRAME_POINTER)
-	if(FLAG_C_FOMIT_FRAME_POINTER)
-		add_compile_options($<$<CONFIG:Release>:-fomit-frame-pointer>)
-	endif()
+	check_and_add_flag(FOMIT_FRAME_POINTER -fomit-frame-pointer RELEASE)
+
+	add_compile_definitions(_DEBUG DEBUG)
+	check_and_add_flag(GGDB -ggdb DEBUG)
```

Comment:
I'm not a big fan of variables with unchecked name like this, seems dangerous and likely to fail sometimes.